src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -98,9 +103,9 @@ protocol SequenceAnnotations {
     }> attributes;
 
     /**
-    Identifiers of children nodes
+    Identifier for the parent annotation, if any.
     */
-    array<string> childrenIds;
+    string parentId;
```

Comment:
Good point, a transcript can be assigned multiple parent genes (e.g. fusion
transcript, trans-spliced, etc)

On Tue, Feb 3, 2015 at 9:01 AM, Uri Laserson <notifications@github.com>
wrote:

> In src/main/resources/avro/sequenceAnnotations.avdl
> <https://github.com/diekhans/ga4gh-schemas-genome-ann/pull/16#discussion_r23990272>
> :
>
> >      */
> > -    array<string> childrenIds;
> > +    string parentId;
>
> +1 to parent over child.
>
> Is there a need to allow an array of parents?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diekhans/ga4gh-schemas-genome-ann/pull/16/files#r23990272>
> .
>